presto-accumulo/src/main/java/com/facebook/presto/accumulo/index/ColumnCardinalityCache.java

```diff
@@ -333,7 +405,7 @@ public Long load(Range key)
          * which occurs when there is no key in Accumulo.
          */
         public class MapDefaultZero
-                extends HashMap<Range, Long>
+                extends HashMap<CacheKey, Long>
```

Comment:
Extending a collection is brittle and error prone, especially in this manner since it violates the map contract and causes inconsistencies between methods like `containsKey()` and `get()`.

I realize this pre-existing, but it would be better to `loadAll()` to fill in zero itself for every key. An easy way is to use `HashMap` and put a zero for every key up front.